src/cursor-doc/token-cursor.ts

```diff
@@ -943,7 +943,8 @@ export class LispTokenCursor extends TokenCursor {
  * Creates a `LispTokenCursor` for walking and manipulating the string `s`.
  */
 export function createStringCursor(s: string): LispTokenCursor {
-  const model = new LineInputModel();
+  const lineEndingSeq = getFirstLineEndingSeq(s);
+  const model = new LineInputModel(lineEndingSeq ? lineEndingSeq.length : 1);
```

Comment:
Looking at this I realize that this probably have been creating weird problems for Windows users for very long. Thanks!

Reply:
Hopefully I can do more testing in the future and discover these pesky windows bugs :)

Reply:
That would be great! We have been lacking someone who can focus on Windows issues a bit.